### PR TITLE
Add encounter editor/theme changes to production

### PR DESCRIPTION
**Pull request by nonowazu**
We're here everyone. We did it. Every *functional* feature we agreed on for EW launch. I want to throw out a huge thanks to every dev that contributed to this to make this PR happen – this site wouldn't exist without you.

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -209,6 +209,76 @@ collections:
...
+        widget: "number"
+      - label: "Tier Weight"
+        name: "tier_weight"
+        hint: (1 - EW, 2 - ShB, 3 - SB, 4 - HW, 5 - ARR)
+        widget: "number"
+      - label: "Coming Soon?"
+        name: "coming_soon"
+        hint: This adds the coming soon overlay. Only enable Coming Soon or Spoilers. Not both.,
+        widget: "boolean"
+        required: false}
```
Comment (OHaiDanny) [inline static/admin/config.yml:257]: @nonowazu the `}` is crashing the editor

**Review by OHaiDanny — CHANGES_REQUESTED** [review]

File: static/admin/config.yml (hunk 1 above)
Comment (nonowazu) [reply static/admin/config.yml:257]: strange that the lint check passes, but I'll fix it.

File: static/admin/config.yml [hunk 2]
```diff
@@ -209,6 +209,76 @@ collections:
...
+        name: "weight"
+        hint: (1 - first, 2 - second, etc.)
+        widget: "number"
+      - label: "Tier Weight"
+        name: "tier_weight"
+        hint: (1 - EW, 2 - ShB, 3 - SB, 4 - HW, 5 - ARR)
+        widget: "number"
+      - label: "Coming Soon?"
+        name: "coming_soon"
+        hint: This adds the coming soon overlay. Only enable Coming Soon or Spoilers. Not both.,
```
Comment (OHaiDanny) [inline static/admin/config.yml:255]: looking at the other lines, do we need that `,` at the end of "Not both."?

File: static/admin/config.yml (hunk 1 above)
Comment (OHaiDanny) [reply static/admin/config.yml]: the error that I got was "required must be a boolean". I think that ) is transforming it into a string

**Review by OHaiDanny — APPROVED** [review]

**Pull request merged**